torch/csrc/jit/codegen/cuda/lower_loops.h

```diff
@@ -44,6 +44,20 @@ class TORCH_CUDA_API LoopNestGenerator : public OptOutDispatch {
   // initialization
   ThreadPredicateMap& thread_predicates_;
 
+  // Fusion shared_memory values
+  // Tracks if shared memory is modified
+  std::unordered_map<Val*, bool> smem_;
+
+  // Clear the modify status for all shared memory buffers
+  void cleanSharedMemory();
+
+  // Toggle modify status for this shared memory buffer
+  void modifySharedMemory(Val* const key);
+
+  // Return the status of the shared memory buffer
+  // False if TensorView is not shared memory buffer
+  bool statusSharedMemory(Val* const key) const;
```

Comment:
`status` seems too generic here: how about a more specific name that would immediately suggest a true/false value, ex. `isSharedMemory` ?

Reply:
I changed it to `isModifiedSharedMemory`